tests/plugins/balloontoolbar/combostate.js

```diff
@@ -0,0 +1,46 @@
+/* bender-tags: balloontoolbar, 1268 */
+/* bender-ckeditor-plugins: balloontoolbar, toolbar, stylescombo */
+
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	var tests = {
+		init: function() {
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 || bender.tools.env.mobile ) {
+				bender.ignore();
```

Comment:
It does not stop the execution of tests – they still run in IE8, ignoring the big alert that tests are ignored in current browser. You should use `assert.ignore` instead.